operators/api/v1alpha2/instance_types.go

```diff
@@ -92,7 +101,9 @@ type InstanceSpec struct {
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
 	// Optional urls for advanced integration features.
-	CustomizationUrls *InstanceCustomizationUrls `json:"customizationUrls,omitempty"`
+	StatusCheckUrl string `json:"statusCheckUrl,omitempty"`
+
+	ContentUrls map[string]*InstanceContentUrls `json:"contentUrls,omitempty"`
```

Comment:
I'm not sure having a map of references is a great idea, a map should be a reference itself already, should be enough